src/components/app-config-ad-groups/index.tsx

```diff
@@ -1,25 +1,50 @@
-import { Radio, TextField, Tooltip, Typography } from '@equinor/eds-core-react';
-import { createRef } from 'react';
+import { Radio, Typography } from '@equinor/eds-core-react';
 import * as PropTypes from 'prop-types';
+import './style.css';
+import { AuthenticatedTemplate } from '@azure/msal-react';
+import {
+  AuthenticationResult,
+  EventMessage,
+  EventType,
+  PublicClientApplication,
+} from '@azure/msal-browser';
+import { msalConfig } from '../graph/Config';
+import { ADGroups } from '../graph/adGroups';
+import { adGroupModel } from '../graph/adGroupModel';
+import { ChangeEvent } from 'react';
 
-import { externalUrls } from '../../externalUrls';
+export type AdGroupsChangeHandler = (event: Event) => void;
 
-import './style.css';
+export interface AppConfigAdGroupsProps {
+  adGroups: string;
```

Comment:
mark optional value optional
```suggestion
  adGroups?: string;
```